### ISSUE TITLE
Add/review resources

### ISSUE THREAD

**Issue by ismayc** [ModernStats/BrainStorming#3]
Maybe identify everything else that is out there where we can take ideas/examples by July 6.  Then we can review resources over the following week or so?

**Comment by rudeboybert** [comment]
Agreed. We need a cutoff or we could fall forever down the rabbit hole.

**Comment by ismayc** [comment]
Sounds good.  Let's plan for 11:59 PM on July 14 to review the resources and get our ideas into the book.  Then we can shoot for writing.  Of course if we are ever in the writing mode on a topic, then we should just go for it.  We can also put stuff in an appendix if we like.
